recipe/meta.yaml

```diff
@@ -87,9 +105,12 @@ outputs:
 {% else %}
         - openmpi {{ openmpi }}                #[mpi_type == 'openmpi']
 {% endif %}
+        # Use pins to control cos6/cos7 match
+        - libgcc-ng  {{ libgcc }}
+        - libstdcxx-ng  {{ libstdcxx }}
       test:
-      imports:
```

Comment:
Was this intentional?

Reply:
Yes, there was an indentation problem earlier. It wasn't an error though.